### PR TITLE
feat(pro): UI updates

### PR DESCRIPTION
**Pull request by olarclara**
## What kind of change does this PR introduce?

- Extends the `Badge` component to accept more props.
- Removes the `Pro` badge.
- Adds the `Free` badge where appropriate.
- Adds the link to team settings.
- Removes unused code + organizes the folder.

## What steps did you take to test this? This is required before we can merge, make sure to test the flow you've updated.

1. Visit `/pro`.

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/7110/commits/d90c196d853159d378a16595578f491093e90332 is at https://pr7110.build.csb.dev/s/new.

File: packages/app/src/app/pages/Pro/components/Switcher.tsx [hunk 1]
```diff
@@ -56,12 +61,25 @@ export const Switcher: React.FC<{
...
+          <Stack css={{ marginLeft: 24 }} direction="vertical">
+            <Stack align="center" gap={1}>
+              <WorkspaceName>
+                <span>{activeTeamInfo.name}</span>
+              </WorkspaceName>
+              {isFreeWorkspace && (
+                <Badge
+                  color="accent"
+                  css={{
+                    height: 'fit-content',
```
Comment (tristandubbeld) [inline packages/app/src/app/pages/Pro/components/Switcher.tsx:73]: Can't check it right now, but `fit-content` isn't supported for Firefox according to MDN, will this cause issues on FF? I'll come back to this if I'm able to test again!

Comment (tristandubbeld) [reply packages/app/src/app/pages/Pro/components/Switcher.tsx:73]: It seems to work, but removing the height style seems to look the same?

**Review by tristandubbeld — APPROVED** [review]

File: packages/app/src/app/pages/Pro/components/Switcher.tsx (hunk 1 above)
Comment (alexnm) [reply packages/app/src/app/pages/Pro/components/Switcher.tsx:73]: I would also avoid passing generic props/css to certain components which should be "closed" like the badge, to ensure we don't abuse this further down the line

**Review by alexnm — APPROVED** [review]
image
Disabled options in the select still have a weird extra padding that needs to be removed, other then that, it looks good. I would suggest some additional changes here, like not even showing the teams that are disabled, but we should merge this first and follow-up with an improvement ticket of the page for later.

**Pull request merged**